src/lib.rs

```diff
@@ -280,10 +317,46 @@ impl<'a> Options<'a> {
     /// **Note:** Only available when the `terminal_size` feature is
     /// enabled.
     #[cfg(feature = "terminal_size")]
-    pub fn with_termwidth() -> Options<'static> {
-        Options::new(termwidth())
+    pub fn with_termwidth() -> Self {
+        Self::new(termwidth())
+    }
+}
+impl<'a, S> Options<'a, S> {
+    /// Creates a new `Options` with the specified width and splitter. Equivalent
```

Comment:
Could you expand the docstring a little to explain _why_ this constructor is necessary or better than using `new()` followed by `splitter()`?

Reply:
Thanks, I see you've updated it!